controller/disaster_revocery_test.go

```diff
@@ -0,0 +1,166 @@
+package controller
```

Comment:
put it in the kubernetes status test, since it's still kubernetes status controller.

Reply:
the TestCaseTemplate is different. The code and logic will be more complex if putting them together

Reply:
Add another test function to kubernetes_controller_test.go . It doesn't need to be the same function.